### PR TITLE
onboard-memory-manager: Add version 1.1.6805

### PR DESCRIPTION
**Pull request by brian6932**
Closes: #9159 
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by issaclin32** [comment]
The **checkver script** is not suitable for official buckets. The bot will need to download the file (about 10MB) every time.

This method is workable for personal buckets, but in official buckets, it will cause out bots to hit the bandwidth limit easily, since we have 1600+ apps to check.

**Comment by brian6932** [comment]
Yea I'm aware, there wasn't really any other way to my knowledge, if someone can find a way to fetch Logitech's versioning without downloading the file, then I'm down to fix that. Or if someone is willing to try to contact someone from Logitech telling them about this issue it would be helpful.

But at the moment, @issaclin32 I see no other way, would you prefer I closed this PR?

**Comment by brian6932** [comment]
Although I will say, the same practice is used for manifests like the Spotify one:
https://github.com/ScoopInstaller/Extras/blob/master/bucket/spotify.json

**Comment by brian6932** [comment]
@issaclin32 The bin is no longer required on the checkver 👍

**Comment by Zliced13** [comment]
@brian6932 do you mind if I do `/verify` just to check?

**Comment by brian6932** [comment]
@rasa The issue was resolved, however, @issaclin32 ddn't respond.
You mind doing it, or at least removing the tags now?

**Comment by Zliced13** [comment]
@HUMORCE could you please approve of this PR? The initial issue at the beginning of the PR has been fixed.

**Comment by HUMORCE** [comment]
I suggest changing the package name to `logitech-omm`.

screenshot_20221024214928

File: bucket/onboard-memory-manager.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.1.6805",
+    "description": "A utility for pro gamers to quickly view, customize, and fine-tune the onboard memory of a compatible Logitech G mouse",
```
Comment (HUMORCE) [inline bucket/onboard-memory-manager.json:3]:
```suggestion
    "description": "OnBoard Memory Manager (OMM) is a utility for pro gamers to quickly view, customize, and fine-tune the onboard memory of a compatible Logitech G mouse.",
```

Comment (HUMORCE) [reply bucket/onboard-memory-manager.json:3]: for SEO if you change the package name to `logitech-omm`.

File: bucket/onboard-memory-manager.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "homepage": "https://support.logi.com/hc/en-us/articles/360059641133",
+    "license": "Freeware",
+    "url": "https://download01.logi.com/web/ftp/pub/techsupport/gaming/Onboard%20Memory%20Manager.exe",
+    "hash": "0abe30ac5946bbd5ceab283ddbcc84f553497b9c1019b28f4d1428461343c4d1",
+    "shortcuts": [
+        [
+            "Onboard%20Memory%20Manager.exe",
+            "Onboard Memory Manager"
```
Comment (HUMORCE) [inline bucket/onboard-memory-manager.json:11]:
```suggestion
            "Onboard Memory Manager.exe",
            "Onboard Memory Manager"
```

Comment (brian6932) [reply bucket/onboard-memory-manager.json:11]: This doesn't work, it is explicitly named: `Onboard%20Memory%20Manager.exe`
I am fine with adding a bin rename to the `preinstall` though.

**Comment by Zliced13** [comment]
@brian6932 sorry for the double /verify, I thought you were done with the manifest.

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**